### DISCUSSION TITLE
[BUG] Can't Make On Ubuntu

### DISCUSSION THREAD

**Discussion by physiii** [svpcom/wfb-ng#175] [General]
Machine:

```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.3 LTS
Release:	20.04
Codename:	focal
```

Here is what I get when running `make deb`.
```
dh: error: unable to load addon python3: Can't locate Debian/Debhelper/Sequence/python3.pm in @INC (you may need to install the Debian::Debhelper::Sequence::python3 module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.30.0 /usr/local/share/perl/5.30.0 /usr/lib/x86_64-linux-gnu/perl5/5.30 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.30 /usr/share/perl/5.30 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at (eval 11) line 1.
BEGIN failed--compilation aborted at (eval 11) line 1.
```

**Comment by svpcom** [comment]
apt-get install dh-python build-essential

**Comment by physiii** [comment]
Awesome, thank you. 

Only other thing I had to do was remove `--install-layout=deb` from the rules file because I was getting `error: option --install-layout not recognized`.

Now I just need figure out how to get all the services/keys running and test it. Great project, hopefully I can get it working!

**Comment by hewittzgh** [comment]
@physiii Hi, I also encountered the same problem as you, "error: option --install-layout not recognized". Can you tell me the detailed solution.Thanks!

**Reply by svpcom** [reply]
apt-get install dh-python debhelper python3-twisted libpcap-dev libsodium-dev python3-pyroute2 python3-future python3-serial
